AllReadyApp/Web-App/AllReady/Controllers/ManageController.cs

```diff
@@ -14,7 +14,33 @@ namespace AllReady.Controllers
     [Authorize]
     public class ManageController : Controller
     {
+        //Email Constants
         private const string EMAIL_CONFIRMATION_SUBJECT = "Confirm your allReady account";
+        private const string NEW_EMAIL_CONFIRM = 
+            "Please confirm your new email address for your allReady account by clicking this link: <a href={0}>link</a>. Note that once confirmed your original email address will cease to be valid as your username.";
+        private const string RESEND_EMAIL_CONFIRM = "Please confirm your allReady account by clicking this link: <a href={0}>link</a>";

```

Comment:
You should wrap the href values in literal quotes (at least to be consistent with what was there before): `<a href=\"{0}\">link</a>`